pkgs/applications/networking/cisco-packet-tracer/customRequireFile.nix

```diff
@@ -0,0 +1,18 @@
+{ requireFile }:
+
+{ filename
+, version
+, sha256
+}: requireFile {
+  name = filename;
+  message = ''
+    Unfortunately, we cannot download the file ${filename} automatically.
```

Comment:
Suggestion:
```
The file '${filename}', version ${version}, is manually downloaded.
You may download it from https://www.netacad.com (or from other sources).

Once downloaded, check the hash:
    sha256sum ${filename}
It should be: ${sha256}

Import it to your nix-store executing the command:
    nix-store --add-fixed sha256 ${filename}

Try to install the package again.
```

Reply:
Because there is no fallback URL anymore I think the default `requireFile` message is good enough